tarpc/src/client.rs

```diff
@@ -22,10 +22,11 @@ use std::{
     fmt,
     pin::Pin,
     sync::{
-        atomic::{AtomicUsize, Ordering},
+        atomic::{AtomicU64, Ordering},
```

Comment:
Changed this to u64 since usize can vary from platform to platform. IF the server and client are on different platforms, this can cause issues, sinc the binary blob will be serialized and deserialized differently. 

This way it results in a compile error at least (if no 64bit atomics are present). 

Compiler guards could be used to make the DefaultSequencer go away if no 64 bit atomics are present, but let's cross that bridge when we get there.